addons/website_event/controllers/main.py

```diff
@@ -149,9 +149,19 @@ def event_page(self, event, page, **post):
         if '.' not in page:
             page = 'website_event.%s' % page
 
+        view = (
+            request.env["website.event.menu"]
+            .sudo()
+            .search([("event_id", "=", event.id), ("view_id.key", "ilike", page)])
+            .view_id
+        )
```

Comment:
Perhaps a bit too much on the multilines here 🙂 Something like this perhaps:
```suggestion
        view = request.env["website.event.menu"].sudo().search([
            ("event_id", "=", event.id), ("view_id.key", "ilike", page)]).view_id
```

Reply:
My bad, it's `Ruff` that automatically format it on save. Updated :+1: